packages/webapp/src/components/Filter/FilterDateRange/index.jsx

```diff
@@ -13,6 +15,7 @@ export function FilterDateRange({
   filterRef,
   shouldReset,
   style,
+  className,
```

Comment:
Thank you 😄 

NIT: Here and elsewhere the className should be added to PropTypes as well